src/Microsoft.DotNet.Cli.Utils/RuntimeConfig.cs

```diff
@@ -32,8 +33,10 @@ public static bool IsApplicationPortable(string entryAssemblyPath)
         } 
  
         private JObject OpenRuntimeConfig(string runtimeConfigPath) 
-        { 
-            return JObject.Parse(File.ReadAllText(runtimeConfigPath)); 
+        {
+            var reader = new JsonTextReader(new StringReader(File.ReadAllText(runtimeConfigPath)));
```

Comment:
Expanding on what @nguerrera JsonTextReader accepts any instance of a type that implements TextReader, @JamesNK (developer of Json.NET) says to use StreamReader (a class implementing TextReader) coupled to a FileStream (either opened explicitly or though StreamReader's constructor).